src/main/java/ProjektiPizza/demo/Security/SecurityConfiguration.java

```diff
@@ -1,48 +0,0 @@
-package ProjektiPizza.demo.Security;
```

Comment:
Why is this deleted? What changed in the code that you could afford deleting this? Can I hit it from the client that is in another IP (is CORS enabled)?

Reply:
we change only first letter Security to security